configure.ac

```diff
@@ -1034,9 +1034,10 @@ mips-sony-bsd|mips-sony-newsos4)
 	CPPFLAGS="$CPPFLAGS -Wextensions -D_XOPEN_SOURCE -D_XOPEN_SOURCE_EXTENDED=1 -Wnowarn=262,1252 -DNSIG=32"
 	AC_DEFINE([HAVE_STRUCT_IN6_ADDR], [1], [struct in6_addr actually defined])
 	AC_DEFINE([HAVE_STRUCT_SOCKADDR_IN6], [1], [struct sockaddr_in6 actually defined])
-	# Todo: Fix this and getrsetbyname.c
-	AC_DEFINE([HAVE__GETSHORT], [2], [incorrect hack to be fixed])
-	AC_DEFINE([HAVE__GETLONG], [2], [incorrect hack to be fixed])
+	AC_DEFINE([HAVE__GETSHORT], [1], [decl exists but impl is missing])
```

Comment:
this doesn't look right to me.   There are existing checks in configure.ac:
                AC_CHECK_FUNCS([_getshort _getlong])
                AC_CHECK_DECLS([_getshort, _getlong], ...

Based on the comments here, I'd without your changes I'd expect it to set HAVE_DECL__GETSHORT=1, HAVE_DECL__GETLONG=1 and not set HAVE__GETSHORT or HAVE__GETLONG.  Is that what it does?  If so the existing #ifndef HAVE__GETLONG and similar should do the right thing.